x-pack/legacy/plugins/apm/public/components/app/ServiceNodeOverview/index.tsx

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { useMemo } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiPanel } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { PROJECTION } from '../../../../common/projections/typings';
+import { LocalUIFilters } from '../../shared/LocalUIFilters';
+import { useUrlParams } from '../../../hooks/useUrlParams';
+import { ManagedTable } from '../../shared/ManagedTable';
+import { useFetcher } from '../../../hooks/useFetcher';
+import { callApmApi } from '../../../services/rest/callApmApi';
+import { asDynamicBytes, asInteger } from '../../../utils/formatters';
+import { ServiceNodeMetricOverviewLink } from '../../shared/Links/apm/ServiceNodeMetricOverviewLink';
+
+const ServiceNodeOverview = () => {
+  const { uiFilters, urlParams } = useUrlParams();
+  const { serviceName, start, end, sortField, sortDirection } = urlParams;
+
+  const localFiltersConfig: React.ComponentProps<
+    typeof LocalUIFilters
+  > = useMemo(
+    () => ({
+      filterNames: ['host', 'containerId', 'podName'],
+      params: {
+        serviceName
+      },
+      projection: PROJECTION.SERVICE_NODES
+    }),
+    [serviceName]
+  );
+
+  const { data: items } = useFetcher(() => {
+    if (!serviceName || !start || !end) {
+      return;
+    }
+    return callApmApi({
+      pathname: '/api/apm/services/{serviceName}/serviceNodes',
+      params: {
+        path: {
+          serviceName
+        },
+        query: {
+          start,
+          end,
+          uiFilters: JSON.stringify(uiFilters),
+          sortField,
+          sortDirection
+        }
+      }
+    });
+  }, [end, serviceName, start, uiFilters, sortField, sortDirection]);
+
+  if (!serviceName) {
+    return null;
+  }
+
+  const columns = [
+    {
+      name: i18n.translate('xpack.apm.jvmsTable.nameColumnLabel', {
+        defaultMessage: 'Name'
+      }),
+      field: 'name',
+      sortable: true,
+      render: (name: string) => {
+        return (
+          <ServiceNodeMetricOverviewLink
+            serviceName={serviceName}
+            serviceNodeName={name}
+          >
+            {name}
+          </ServiceNodeMetricOverviewLink>
+        );
+      }
+    },
+    {
+      name: i18n.translate('xpack.apm.jvmsTable.cpuColumnLabel', {
+        defaultMessage: 'CPU'
+      }),
+      field: 'cpu',
+      sortable: true,
+      render: asDynamicBytes
```

Comment:
I think we should add `number` as `dataType` to make sure numbers are right-aligned.

Reply:
oh, woops, this should be `asPercent`, right? Thanks for pointing it out